src/server/index.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const os = require('os');
+
+const app = express();
+
+app.use(express.static('dist'));
+
+// Index endpoint
+app.get('/', (req, resp) => resp.send('Hello World'));
+
+// Example endpoint
+app.get('/api/getUsername', (req, res) =>
```

Comment:
Please use kebab case for apis or path /api/user/name. Please do not use camel case for api endpoints.